API/Controllers/ViewMemoryController.js

```diff
@@ -0,0 +1,64 @@
+import { response } from 'express';
+
+class ViewMemoryController {
+
+    static FetchViewMemoryPage(request, response){
```

Comment:
```suggestion
    static FetchMemory(request, response){
```
We are not returning a page, but a response in JSON format

Reply:
your methods have to be camelCase